src/components/inputs/text-input/README.md

```diff
@@ -22,17 +22,15 @@ import { TextInput } from '@commercetools-frontend/ui-kit';
 | `onChange`             | `func`   |    -     | -                            | -       | Called with the new value. Required when input is not read only. Parent should pass it back as `value` |
-| `onBlur`               | `func`   |    -     | -                            | -       | Called when field is blurred                                                                           |
-| `onFocus`              | `func`   |    -     | -                            | -       | Called when field is focused                                                                           |
-| `isAutofocussed`       | `bool`   |    -     | -                            | -       | Focus the input field on initial render                                                                |
-| `isDisabled`           | `bool`   |    -     | -                            | `false` | Indicates that the field cannot be used (e.g not authorised, or changes not saved)                     |
-| `isReadOnly`           | `bool`   |    -     | -                            | `false` | Indicates that the field is displaying read-only content                                               |
+| `autoFocus`            | `bool`   |    -     | -                            | -       | Focus the input field on initial render                                                                |
+| `disabled`             | `bool`   |    -     | -                            | `false` | Indicates that the field cannot be used (e.g not authorised, or changes not saved)                     |
+| `readOnly`             | `bool`   |    -     | -                            | `false` | Indicates that the field is displaying read-only content                                               |
 | `hasError`             | `bool`   |    -     | -                            | -       | Indicates the input field has an error                                                                 |
 | `hasWarning`           | `bool`   |    -     | -                            | -       | Indicates the input field has a warning                                                                |
 | `placeholder`          | `string` |    -     | -                            | -       | Placeholder text for the input                                                                         |
-| `horizontalConstraint` | `object` |          | `s`, `m`, `l`, `xl`, `scale` | `scale` | Horizontal size limit of the input fields.                                                             |
+| `horizontalConstraint` | `object` |          | `s`, `m`, `l`, `xl`, `scale` | `scale` | Horizontal size limit of the wrapper for input fields.                                                 |
 
-The component further forwards all `data-` attributes to the underlying `input` component.
+Any other properties will be spread to the underlying `input` element.
```

Comment:
I'm not sure if we should keep the documentation for the props we already had, but I'm fine with it.